public/robots.txt

```diff
@@ -0,0 +1,2 @@
+User-agent: *
+Disallow: /
```

Comment:
Out of curiosity, why this config?

Reply:
This is like the de facto allow everything robots.txt

Reply:
actually, it isn't. Good call, need to remove the `/`